Numsense/ObjectOriented.fs

```diff
@@ -43,10 +43,17 @@ type RussianNumeralConverter () =
         member this.ToNumeral number = Numeral.toRussian number
         member this.TryParse (s, result) =
             Helper.tryParse Numeral.tryParseRussian (s, &result)
+
+type LithuanianNumeralConverter () =
```

Comment:
Using the Devil's Advocate review technique, if I remove this type, then all tests still pass.